app/src/main/java/com/onboarding/pronosticAR/utils/ConstantUtils.kt

```diff
@@ -4,4 +4,36 @@ object ConstantUtils {
     const val DEFAULT_INT_VALUE: Int = 0
     const val DEFAULT_DOUBLE_VALUE: Double = 0.0
     const val DEFAULT_STRING_VALUE: String = "none"
+    const val NORTH: String = "N"
+    const val SOUTH: String = "S"
+    const val EAST: String = "E"
+    const val WEST: String = "O"
+    const val NORTHEAST: String = "NE"
+    const val NORTHWEST: String = "NO"
+    const val SOUTHEAST: String = "SE"
+    const val SOUTHWEST: String = "SO"
+    const val PERCENTAGE_ADJUST: Int = 100
+    const val SLASH: String = "/"
+    const val COLON: String = ":"
+    const val STRING_DAY_START: Int = 8
+    const val STRING_DAY_END: Int = 10
+    const val STRING_MONTH_START: Int = 5
+    const val STRING_MONTH_END: Int = 7
+    const val STRING_HOUR_START: Int = 11
+    const val STRING_HOUR_END: Int = 13
+    const val STRING_MINUTES: String = "00"
+    const val NORTHEAST_START: Int = 21
+    const val NORTHEAST_END: Int = 69
+    const val EAST_START: Int = 70
+    const val EAST_END: Int = 110
+    const val SOUTHEAST_START: Int = 111
+    const val SOUTHEAST_END: Int = 159
+    const val SOUTH_START: Int = 160
+    const val SOUTH_END: Int = 200
+    const val SOUTHWEST_START: Int = 201
+    const val SOUTHWEST_END: Int = 249
+    const val WEST_START: Int = 250
+    const val WEST_END: Int = 290
+    const val NORTHWEST_START: Int = 291
+    const val NORTHWEST_END: Int = 339
```

Comment:
why are these consts here if they are only used in one function?

Reply:
consts moved